src/main/java/com/mitchej123/hodgepodge/core/HodgepodgeMixinPlugin.java

```diff
@@ -168,7 +168,7 @@ public void postApply(String targetClassName, ClassNode targetClass, String mixi
         ),
         WAKE_ANCHORS_ON_LOGIN("Wake up personal anchors on owner login",
             () -> config.installAnchorAlarm,
-            "Railcraft_1.7.10",
+            "Railcraft-1.7.10",
```

Comment:
Huh, that's why archor wakeup broke..

Reply:
Yah... I'm going to add a config option to make jars required and have it exit if they're not found.